swan-lake/integration-technology-tools/health-tool.md

```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
 title: Health tool (FHIR/HL7)
 description: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 keywords: ballerina, programming language, health, contract
-permalink: /learn/edi-tool/
+permalink: /learn/health-tool/
 active: health-tool
 intro: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 ---
+
+The Health tool can generate accelerators primarily focussed on the [Fast Healthcare Interoperability Resources (FHIR) standard](https://www.hl7.org/fhir/overview.html). The Health tool is supported from Ballerina version 2201.7.0 (Swan Lake Update 7) onwards.
+
+## Ballerina FHIR package generator
+
+This tool is available in the latest version of the Health tool (v1.0.0). It generates a Ballerina package for a given FHIR implementation guide. The FHIR resources in the implementation guide will be represented as Ballerina records including the correct cardinality constraints and metadata.
+
+FHIR integration developers can leverage the generated package when transforming custom health data into FHIR format without referring to the specification.
+
+## Prerequisites
+
+- [Ballerina 2201.7.0 (Swan Lake Update 7)](https://ballerina.io/downloads/)
+- A directory containing all FHIR specification definition files.
+
+    >**Tip:** You can download a preferred standard implementation guide from the [FHIR Implementation Guide registry](http://fhir.org/guides/registry/).
```

Comment:
It is not that good to have `Tip` and `Note` both together. Seems tip line can be merged to `A directory containing all FHIR specification definition files.` it self.

Reply:
Merged.